tests/javascripts/templateFolderForm.test.js

```diff
@@ -309,6 +309,36 @@ describe('TemplateFolderForm', () => {
 
   });
 
+  describe("Click 'New template' for single channel service", () => {
+    test("should redirect to new template page", () => {
+      setFixtures(hierarchy, "data-channel='sms' data-service='123'")
```

Comment:
You need to re-declare the value of `templateFolderForm` after calling `setFixtures`, otherwise it refers to elements which are no longer in the page:

```suggestion
      setFixtures(hierarchy, "data-channel='sms' data-service='123'")
      templateFolderForm = document.querySelector('form[data-module=template-folder-form]');
```